.github/CODEOWNERS

```diff
@@ -151,3 +151,11 @@
 
 /odoo/http.py @odoo/rd-security @odoo/rd-website
 /odoo/sql_db.py @odoo/rd-security
+
+/addons/bus @odoo/rd-framework-js
+/addons/web/ @odoo/rd-framework-js
+/addons/web_kanban_gauge/ @odoo/rd-framework-js
+/odoo/addons/base/models/assetsbundle.py @odoo/rd-framework-js
+/odoo/cli/ts_config.py @odoo/rd-framework-js
+/odoo/tools/js_transpiler.py @odoo/rd-framework-js
+/odoo/tools/sourcemap_generator.py @odoo/rd-framework-js
```

Comment:
Hello, order is this file is very important (last matching rule has priority), so we enforce rather strict guidelines (see comments at the top of the file).

In this case, they should be sorted alphabetically among other modules after the `# Specific rules` line.